src/ui/pages/SendTransaction/SendTransaction.tsx

```diff
@@ -230,7 +232,11 @@ function TransactionDefaultView({
       });
     },
     onMutate: () => 'sendTransaction',
-    onSuccess: (tx) => handleSentTransaction(tx),
+    onSuccess: (tx) =>
+      setTimeout(
+        () => handleSentTransaction(tx),
+        preferences?.showHoldToSignButton ? 500 : 0
```

Comment:
As I understand, we want to show success state for a second to the user before closing the window